cloudflare/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:jessie
+
+# install curl and jq
+RUN apt-get update && \
+    apt-get install -y \
+    curl \
+    jq && \
+    rm -rf /var/lib/apt/lists/*
+
+# add containerbuddy and configuration
+ADD opt/containerbuddy /opt/containerbuddy/
```

Comment:
There's a strong argument against using [`ADD`](https://docs.docker.com/reference/builder/#add) unless you want to specifically use it's features over [`COPY`](https://docs.docker.com/reference/builder/#copy)